recipes/gatk-framework/gatk-framework

```diff
@@ -6,17 +6,7 @@
 set -o pipefail
 export LC_ALL=en_US.UTF-8
 
-# Find original directory of bash script, resovling symlinks
-# http://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in/246128#246128
-SOURCE="${BASH_SOURCE[0]}"
-while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
-    DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-    SOURCE="$(readlink "$SOURCE")"
-    [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
-done
-DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-
-JAR_DIR=$DIR
+JAR_DIR="$(cd "$(dirname "$0")" && pwd -P)"
```

Comment:
I don't immediately get this. Can you explain why this can replace the code on the left?